spec/controllers/chapters_controller_spec.rb

```diff
@@ -473,6 +473,15 @@
             expect(assigns[:chapter].posted).to be true
           end
 
+          it "updates cached chapter counts" do
+            work.invalidate_work_chapter_count(work)
```

Comment:
You should be able to remove the `invalidate_work_chapter_count` calls from the tests, as they should be correctly called by the controller.

Reply:
Done in https://github.com/otwcode/otwarchive/pull/4159/commits/2fac5f9549464472387c72e0c29721551e08ec78